ui/lib/pki/addon/components/page/pki-overview.hbs

```diff
@@ -64,22 +64,21 @@
         @fallbackComponent="input-search"
         data-test-view-certificate-input
       />
-      <button
-        type="button"
-        class="button is-secondary {{unless this.certificateValue 'has-left-margin-s'}}"
+      <Hds::Button
+        @text="View"
+        @color="secondary"
+        class={{unless this.certificateValue "has-left-margin-s"}}
```

Comment:
I think this bumps the button to the other line when it has a certificate, so unless you remove `is-marginless` this button might be super close to the search select input